.github/workflows/tests.yml

```diff
@@ -168,4 +168,4 @@ jobs:
           maven-version: 3.9.0
       # we dont enforce that the version must be non snapshot as this is not possible for SNAPSHOT versions in our workflow.
       - name: Check maven pom quality
-        run: mvn org.kordamp.maven:pomchecker-maven-plugin:1.9.0:check-maven-central -D"checker.release=false"
+        run: mvn -f spoon-pom org.kordamp.maven:pomchecker-maven-plugin:1.9.0:check-maven-central -D"checker.release=false"
```

Comment:
Does this check also run for all submodules now?

Reply:
Yes, but there are only two: spoon-javadoc and spoon-core.